windows_containers/windows-containers-release-notes-8-x.adoc

```diff
@@ -28,6 +28,20 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-8-1-2"]
+== Release notes for Red Hat Windows Machine Config Operator 8.1.2
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of WMCO 8.1.2 were released in link:https://access.redhat.com/errata/RHBA-2023:{TBD}[RHSA-{TDB}].
+
+[id="wmco-8-1-2-bug-fixes"]
+=== Bug fixes
+
+// Also in 10.15.0
+* Previously, because of bad logic in the networking configuration script, the WICD was incorrectly reading carriage returns in the CNI configuration file as changes, and identified the file as modified. This caused the CNI configuration to be unnecessarily reloaded, potentially resulting in container restarts and brief network outages. With this fix, the WICD now reloads the CNI configuration only when the CNI configuration is actually modified. (link:https://issues.redhat.com/browse/OCPBUGS-27046[*OCPBUGS-27046*])
+
+// Also in 10.15.0
+* Previously, because of a lack of synchronization between Windows machine set nodes and BYOH instances, during an upgrade the machine set nodes and the BYOH instances could upgrade simultaneously. This could impact running workloads. This fix introduces a locking mechanism so that machine set nodes and BYOH instances upgrade individually. (link:https://issues.redhat.com/browse/OCPBUGS-23016[*OCPBUGS-23016*])
```

Comment:
Update is the preferred term [per our term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update): 
```suggestion
* Previously, because of a lack of synchronization between Windows machine set nodes and BYOH instances, during an update the machine set nodes and the BYOH instances could update simultaneously. This could impact running workloads. This fix introduces a locking mechanism so that machine set nodes and BYOH instances update individually. (link:https://issues.redhat.com/browse/OCPBUGS-23016[*OCPBUGS-23016*])
```